src/content/developers/docs/consensus-mechanisms/pos/attack-and-defense/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Ethereum proof-of-stake attack and defense
+description: Learn about the known attack vectors on proof-of-stake Ethereum and how they are defended.
+lang: en
+---
+
+Thieves and saboteurs are constantly seeking opportunities to attack Ethereum’s client software. This page outlines the known attack vectors on Ethereum’s consensus layer and outlines how those attacks can be defended. The information on this page is adapted from a [longer form version](https://mirror.xyz/jmcook.eth/YqHargbVWVNRQqQpVpzrqEQ8IqwNUJDIpwRP7SS5FXs).
+
+## Prerequisites {#prerequisites}
+
+Some basic knowledge of proof-of-stake is required. Also, it will be helpful to have a basic understanding of Ethereum's [incentive layer](/docs/consensus-mechanisms/pos/rewards-and-penalties) and fork-choice algorithm, [LMD-GHOST](/docs/consensus-mechanisms/pos/gasper).
```

Comment:
```suggestion
Some basic knowledge of [proof-of-stake](/developers/docs/consensus-mechanisms/pos/) is required. Also, it will be helpful to have a basic understanding of Ethereum's [incentive layer](/docs/consensus-mechanisms/pos/rewards-and-penalties) and fork-choice algorithm, [LMD-GHOST](/docs/consensus-mechanisms/pos/gasper).
```